src/Model/SoftDeletesTrait.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Mongolid\Model;
+
+use DateTime;
+use MongoDB\BSON\UTCDateTime;
+use Mongolid\Cursor\CursorInterface;
+use Mongolid\Util\QueryBuilder;
+
+trait SoftDeletesTrait
+{
+    public bool $enabledSoftDeletes = true;
```

Comment:
```suggestion
    protected bool $isSoftDeleteEnabled = true;
```
Maybe we can change this attribute to protected. What do you think?